gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -73,20 +77,36 @@ func NewVMKeeper(
 	return vmk
 }
 
-func (vm *VMKeeper) Initialize(ms store.MultiStore) {
+func (vm *VMKeeper) Initialize(logger *slog.Logger, ms store.MultiStore) {
```

Comment:
A way to speed up non-production runtimes, such as txtar testing, would be to add a flag that allows switching between the two modes: loading everything or still using the package getter. This approach would enable the use of the package getter everywhere, while maintaining at least one CI test that utilizes the new loading mechanism, such as a txtar command `preload_stdlibs`.

Reply:
I'm going to try applying the diff I started above (see OP), and try to split it so that we're sure to have a database with only the packages in there (and no other additional data). don't know if it can work; but I'd want to avoid testscript from straying too much from the "real" behaviour